hooks/useNavigationForm.js

```diff
@@ -45,13 +45,15 @@ const useNavigationForm = (jsonFile, preview) => {
       component: "group-list",
       description,
       itemProps: (item) => ({
+        key: item.id,
         label: `${item.title} (${item.type})`,
       }),
       defaultItem: () => ({
         type: "link",
         slug: "getting-started",
         title: "new doc page",
         children: [],
+        id: Math.random().toString(36).substr(2, 9),
```

Comment:
Might as well toss this into a utility function, just in case we ever want to use it anywhere else or ever want to switch this to use UUIDs or something.

Other than that, this fixed the issue!

Reply:
@mittonface Good catch I also forgot to add the ID to the content-creator plugin. fixed that as well